internal/server/data/migrator/migrator.go

```diff
@@ -177,7 +190,7 @@ func (g *Migrator) runInitSchema() error {
 			return err
 		}
 	}
-	return nil
+	return g.options.LoadKey(g.tx)
```

Comment:
Why does this get called twice?

Reply:
There are 3 possible flows for migrations:
1. fresh install - call initialize schema and write migration IDs to the migrations table
2. run new migrations
3. don't run any migrations if they have all been applied

One call is for flow 1, and second call is for flow 2 and 3.